### PR TITLE
[datadog_logs_metric] Add include_percentiles attribute to distribution compute

### PR DESCRIPTION
**Pull request by JeanCoquelet**
See https://datadoghq.atlassian.net/browse/LA-2546

We added `include_percentiles` in Logs Metrics public API as it was missing from the public API but present internally.

### CODE REVIEW

**Review by nkzou — APPROVED** [review]

File: docs/resources/logs_metric.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Required:
 
 Optional:
 
+- `include_percentiles` (Boolean) Toggle to include/exclude percentiles for a distribution metric. Defaults to false. Can only be applied to metrics that have a `aggregation_type` of distribution.
```
Comment (cswatt) [inline docs/resources/logs_metric.md:62]:
```suggestion
- `include_percentiles` (Boolean) Toggle to include/exclude percentiles for a distribution metric. Defaults to false. Can only be applied to metrics that have an `aggregation_type` of distribution.
```

**Review by cswatt — COMMENTED** [review]
one nit, otherwise looks good for docs!

File: datadog/tests/resource_datadog_logs_metric_test.go [hunk 2]
```diff
@@ -82,8 +82,9 @@ func testAccCheckDatadogLogsMetricConfigBasic(uniq string) string {
         resource "datadog_logs_metric" "testing_logs_metric" {
 			name = "%s"
 			compute {
-				aggregation_type = "distribution"
-				path             = "@duration"
+				aggregation_type    = "distribution"
+				path                = "@duration"
+				include_percentiles = false
```
Comment (corentinmarc) [inline datadog/tests/resource_datadog_logs_metric_test.go:87]: Can we update the test `TestAccDatadogLogsMetric_Basic` as well to check the terraform resource state contain the `include_percentiles` value ?
Like https://github.com/DataDog/terraform-provider-datadog/blob/fd929126ea0cfd8e86605d3d12fc63be4a040e35/datadog/tests/resource_datadog_logs_metric_test.go#L72

File: docs/resources/logs_metric.md (hunk 1 above)
Comment (JeanCoquelet) [reply docs/resources/logs_metric.md]: Good catch! Updated here 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/88396003d96590c71f9ea39c718a71015f8c7d80)

File: datadog/tests/resource_datadog_logs_metric_test.go (hunk 2 above)
Comment (JeanCoquelet) [reply datadog/tests/resource_datadog_logs_metric_test.go:89]: 🤦 I removed it and forgot to put it back. Updated here 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/e3881747942922254a069690327ce144c0716ea4)

File: docs/resources/logs_metric.md (hunk 1 above)
Comment (JeanCoquelet) [reply docs/resources/logs_metric.md]: ... And here, where it should have been updated in the first place 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/117c204b19f5dfdcea6b81917fddb41dc8b7c937)

File: datadog/tests/resource_datadog_logs_metric_test.go (hunk 2 above)
Comment (corentinmarc) [reply datadog/tests/resource_datadog_logs_metric_test.go:89]: Thanks ! 🙇‍♂️

**Review by corentinmarc — DISMISSED** [review]
LGTM

File: datadog/tests/cassettes/TestAccDatadogLogsMetric_Basic.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ version: 1
 interactions:
 - request:
     body: |
-      {"data":{"attributes":{"compute":{"aggregation_type":"distribution","path":"@duration"},"filter":{"query":"service:test"},"group_by":[{"path":"@my.status","tag_name":"status"},{"path":"service","tag_name":"service"}]},"id":"tf_TestAccDatadogLogsMetric_Basic_local_1615587158","type":"logs_metrics"}}
+      {"data":{"attributes":{"compute":{"aggregation_type":"distribution","include_percentiles":false,"path":"@duration"},"filter":{"query":"service:test"},"group_by":[{"path":"@my.status","tag_name":"status"},{"path":"service","tag_name":"service"}]},"id":"tf_TestAccDatadogLogsMetric_Basic_local_1669226251","type":"logs_metrics"}}
```
Comment (camilledolle) [inline datadog/tests/cassettes/TestAccDatadogLogsMetric_Basic.yaml:6]: Could we add a test for the `PATCH` with a change of the `include_percentiles`? There is similar test [here](https://github.com/DataDog/terraform-provider-datadog/blob/e15155c75c87900d52f09bc467afd2b6f4f09577/datadog/tests/cassettes/TestAccDatadogMetricTagConfiguration_Basic.yaml#L97) for standard metrics

Comment (JeanCoquelet) [reply datadog/tests/cassettes/TestAccDatadogLogsMetric_Basic.yaml]: Added patch test case here 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/6bbb1e072a7354cd6d021cdff098740c5087f143)

**Review by nkzou — APPROVED** [review]

**Review by camilledolle — APPROVED** [review]

**Review by corentinmarc — APPROVED** [review]

**Pull request merged**